sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -21,7 +21,7 @@ This package contains a Java SDK for Azure Communication CallingServer Service.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-communication-callingserver</artifactId>
-    <version>1.0.0-beta.1</version>
+    <version>1.0.0-beta.2</version>
```

Comment:
Why do you bump up version here? Was beta.1 released already? If So, modify ChangeLog.md to reflect the version changes.

Reply:
Updated changelog file.